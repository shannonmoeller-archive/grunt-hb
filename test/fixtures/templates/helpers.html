<!doctype html>
<html>
<head>
	<title>Helpers</title>
</head>
<body>
	{{upper "Hello World"}}
	{{flow-when "a" "b"}}
</body>
</html>
