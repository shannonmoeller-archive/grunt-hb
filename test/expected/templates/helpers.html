<!doctype html>
<html>
<head>
	<title>Helpers</title>
</head>
<body>
	HELLO WORLD
	false
</body>
</html>
